### PR TITLE
User story tracking: repo setup 

### PR DESCRIPTION
**Pull request by astoycos**
This PR makes the template that user stories should follow and makes a new directory for user stories related to ClusterNetworkPolicy

### CODE REVIEW

**Comment by astoycos** [comment]
@Dyanngg This is what I was thinking for housing of any user stories for CNP let me know what you think

**Comment by astoycos** [comment]
Updated @Dyanngg

**Review by Dyanngg — APPROVED** [review]
lgtm, thanks @astoycos

File: docs/userstories/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+[Some complementary notes if necessary:]
+
+* > Here goes a quote from an email
+* Here goes whatever useful information can exist…
+```
+
+### Resources:
+
+* [Style-guides and template for a user story](agile-user-story.md)
```
Comment (srampal) [inline docs/userstories/README.md:62]: Markdown formatting error ? The link didnt work for me. Should be formatted as \[text\]\(url\)

**Review by srampal — COMMENTED** [review]
Looks good.
1) Looks like there is a markdown formatting error in the very last section (References). The link does not work. It needs to be formatted as [text](url). Please check and fix.
2) This template should be more of a  suggested guideline

File: docs/userstories/README.md (hunk 1 above)
Comment (astoycos) [reply docs/userstories/README.md]: Fixed Thanks!

**Comment by astoycos** [comment]
Thanks for the catch! Fixed the error and added a line explaining that the template is just a "guideline"

**Review by srampal — COMMENTED** [review]
Please remove that file package.json that has showed up in the diffs

**Comment by astoycos** [comment]
Sorry about that @srampal It's done

**Pull request merged**